std/math.d

```diff
@@ -6832,6 +6832,141 @@ if (isFloatingPoint!(F) && isFloatingPoint!(G))
     assert(approxEqual(pow(twoI, three), 8.0));
 }
 
+/** Computes the value of a positive integer x, raised to the power n, modulo m.
+ *
+ * All values have to be unsigned types, otherwise it will raise a compilation error.
+ */
+Unqual!(Largest!(F, H)) powmod(F, G, H)(F x, G n, H m)
+if (isIntegral!F && isIntegral!G && isIntegral!H &&
+    isUnsigned!F && isUnsigned!G && isUnsigned!H)
+{
+    import std.traits : AliasSeq;
```

Comment:
`AliasSeq` is defined in `std.meta` - change the import to: `import std.meta : AliasSeq;`